.github/workflows/xe-experimental-vm-test.yml

```diff
@@ -0,0 +1,35 @@
+name: "[experimental] VM-based integration testing, ping @Xe if it fails"
```

Comment:
long name is long :)

Everything we have is experimental to some degree, and people can ping #eng if anythign fails they don't understand, so just "integration-vms" is a good short name.